composer.json

```diff
@@ -8,9 +8,18 @@
             "email": "szymon.kulczynski@sylius.com"
         }
     ],
+    "repositories": [
```

Comment:
Why is this  necessary?

Reply:
Because, this package is not on Packagist : https://packagist.org/search/?q=sylius/shop-bundle
So composer install not found it.

Reply:
I don't know of any cases that use the shop bundle without Sylius and I don't think we need to support that case either. Maybe it would be better to change the dependency to `sylius/sylius` (containing the shop bundle).

Reply:
I agree with @4c0n. We need ShopBundle for a theme but ShopBundle is in `sylius/sylius:^1.7`.

Reply:
I've changed it, thanks.